wpilibj/src/test/java/edu/wpi/first/wpilibj/geometry/GeometryTest.java

```diff
@@ -0,0 +1,70 @@
+package edu.wpi.first.wpilibj.geometry;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class GeometryTest {
+
+  private static double kEpsilon = 1E-9;
+
+  /**
+   * Asserts "val1" is within "eps" of "val2".
+   *
+   * @param val1 First operand in comparison.
+   * @param val2 Second operand in comparison.
+   * @param eps Tolerance for whether values are near to each other.
+   */
+  private static void assertNear(double val1, double val2, double eps) {
+    assertTrue(Math.abs(val1 - val2) <= eps, "Difference between " + val1 + " and " + val2
+        + " is greater than " + eps);
+  }
```

Comment:
Replace assertEquals.  It has an override that does this.

Reply:
Began to use `assertEquals` override in 7fc4c4f